propagation/trace_context_test.go

```diff
@@ -277,17 +278,82 @@ func TestTraceContextPropagator_GetAllKeys(t *testing.T) {
 
 func TestTraceStatePropagation(t *testing.T) {
 	prop := propagation.TraceContext{}
-	want := "opaquevalue"
-	headerName := "tracestate"
+	stateHeader := "tracestate"
+	parentHeader := "traceparent"
+	state, _ := trace.TraceStateFromKeyValues(label.String("key1", "value1"), label.String("key2", "value2"))
```

Comment:
We should probably fail here if there is a returned error. That way if future changes make this call fail it will be easier to debug.